Backend/config/database.py

```diff
@@ -2,9 +2,10 @@
 
 MongoClient()
 
-client = MongoClient(
-    "mongodb+srv://username:password@cluster0.mbx9gz6.mongodb.net/?retryWrites=true&w=majority")
-db = client.Malnutrition
-NgoCollection = db["Ngo"]
-UserCollection = db["User"]
+client = MongoClient("mongodb+srv://AlekhyaA:alli2414@cluster0.cythueh.mongodb.net/?retryWrites=true&w=majority")
+#db = client.Malnutrition
+#NgoCollection = db["Ngo"]
+#UserCollection = db["User"]
+db =client["ngo_login"]
```

Comment:
let us not comment these lines of db and collection because they are used. What you can do use is use the same DB and create a new collection donors enough